SQLQuery_ZooManagement.sql

```diff
@@ -0,0 +1,164 @@
+CREATE Database ZooManagement;
+
+Use ZooManagement;
+
+CREATE TABLE Species
+(
+Id INT IDENTITY(1,1) PRIMARY KEY,
+Species VARCHAR(150),
+SpeciesNumber INT,
+AnimalsNumber INT,
+);
+
+CREATE TABLE Visitors
+(
+Id INT IDENTITY(1,1) PRIMARY KEY,
+Day VARCHAR(150),
+NumberOfVisitors INT,
+);
+
+CREATE TABLE Tickets
+(
+Id INT IDENTITY(1,1) PRIMARY KEY,
+Price INT,
+Validity VARCHAR(150),
+);
+
+CREATE TABLE Caretakers
+(
+Id INT IDENTITY(1,1) PRIMARY KEY,
+Name VARCHAR(150),

```

Comment:
The caretacker should also hold information about the zoo he / she works for. So a ZooId referencing a Zoos table would be good to have.